docs/configuration/schema-config-reference.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Schema Configuration"
+linkTitle: "Schema Configuration"
+weight: 1
+slug: schema-configuration
+---
+
+Cortex uses a NoSQL Store to store its index and optionally an Object store to store its chunks. Cortex has overtime evolved its schema to be more optimal and better fit the use-cases that arose. Currently there are 9 schemas that are used in production but we recommend running with `v9` schema when possible. You can move from one schema to another if a new schema fits your purpose better, but you still need to configure Cortex to make sure it can read the old data in the old schemas.
```

Comment:
```suggestion
Cortex uses a NoSQL Store to store its index and optionally an Object store to store its chunks. Cortex has overtime evolved its schema to be more optimal and better fit the use cases that arose.

Currently there are 9 schema versions that are used in production but we recommend running with `v9` schema when possible. You can move from one schema to another if a new schema fits your purpose better, but you still need to configure Cortex to make sure it can read the old data stored with the old schemas.
```